app/assets/javascripts/admin/addon/models/report.js

```diff
@@ -407,12 +407,21 @@ export default class Report extends EmberObject {
 
   changeTitle(valAtT1, valAtT2, prevPeriodString) {
     const change = this.percentChangeString(valAtT1, valAtT2);
```

Comment:
I'd also have a closer look at the `percentChangeString` method. Could `js.number.percent` be used instead of hard-coding the percent sign?